### PR TITLE
Zend Framework 1 integration

### PR DESCRIPTION
**Pull request by SammyK**
### Description

This PR adds integration support for [Zend Framework 1](https://framework.zend.com/manual/1.12/en/manual.html).

### Readiness checklist
- [x] [Changelog entry](docs/changelog.md) added, if necessary
- [x] Tests added for this feature/bug

### CODE REVIEW

File: src/DDTrace/Integrations/ZendFramework/V1/TraceRequest.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    /**
+     * @param Zend_Controller_Request_Abstract|Zend_Controller_Request_Http $request
+     */
+    public function postDispatch(Zend_Controller_Request_Abstract $request)
+    {
+        $scope = GlobalTracer::get()->getRootScope();
+        if (null === $scope) {
+            return;
+        }
+        $scope->getSpan()->setTag('http.status_code', $this->getResponse()->getHttpResponseCode());
```
Comment (labbati) [inline src/DDTrace/Integrations/ZendFramework/V1/TraceRequest.php]: Can we use the constant `Tag::HTTP_STATUS_CODE` here? :)

File: src/DDTrace/Integrations/ZendFramework/V1/TraceRequest.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            return;
+        }
+        $span = $scope->getSpan();
+        $span->setTag('zf1.controller', $request->getControllerName());
+        $span->setTag('zf1.action', $request->getActionName());
+        $span->setTag(
+            'zf1.route_name',
+            Zend_Controller_Front::getInstance()->getRouter()->getCurrentRouteName()
+        );
+        $span->setTag('http.method', $request->getMethod());
```
Comment (labbati) [inline src/DDTrace/Integrations/ZendFramework/V1/TraceRequest.php]: Can we use the constants `Tag::HTTP_URL` and `Tag::HTTP_METHOD` here? :)

File: tests/Frameworks/ZendFramework/Version_1_12/application/controllers/SimpleController.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    {
+        // Don't auto render this action
+        $this->_helper->viewRenderer->setNoRender();
+        echo 'This is a string.';
+    }
+
+    public function viewAction()
+    {
+        // Empty
+    }
```
Comment (labbati) [inline tests/Frameworks/ZendFramework/Version_1_12/application/controllers/SimpleController.php]: Where are the methods for view rendering and exception  testing :) ?

Comment (SammyK) [reply tests/Frameworks/ZendFramework/Version_1_12/application/controllers/SimpleController.php]: This one renders the view in `application/views/scripts/simple/view.phtml`. ZF1 does a lot of magic to render views. And the exception case is found in the `ErrorController.php`. :)

**Pull request merged**